src/Voron/Impl/FileHeaders/MetadataAccessor.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Sparrow;
+using Voron.Global;
+
+namespace Voron.Impl.FileHeaders;
+
+public delegate void ModifyMetadataAction(ref MetadataFile header);
+
+public sealed class MetadataAccessor(StorageEnvironment env)
+{
+    internal static string MetadataName = "metadata.one";
```

Comment:
Why `metadata.one`? There won't be a `metadata.two`, right?
`metadata.id` would probably be better?
Or `database.metadata` ?